UnityClient/Assets/Scenes/CharSelection/CharSelectionController.cs

```diff
@@ -133,4 +136,52 @@ public class CharSelectionController : MonoBehaviour {
             CharNum = (byte) currentCharactersInfo.Chars.Count
         }.Send();
     }
+    
+    private void SetTextFields(CharacterData character) {
+        Text[] fields = textPanel.GetComponentsInChildren<Text>();
+        int i = 0;
+        foreach (Text field in fields) {
```

Comment:
I think you can still use a for instead of foreach, it is easy to lose track of the `i++` at the bottom

Reply:
Ok, changed :)